Swiftfin/Views/VideoPlayer/VLCPlayerViewController.swift

```diff
@@ -205,22 +230,24 @@ class VLCPlayerViewController: UIViewController {
 
 		let singleTapGesture = UITapGestureRecognizer(target: self, action: #selector(didTap))
 
-		let rightSwipeGesture = UISwipeGestureRecognizer(target: self, action: #selector(didRightSwipe))
-		rightSwipeGesture.direction = .right
-
-		let leftSwipeGesture = UISwipeGestureRecognizer(target: self, action: #selector(didLeftSwipe))
-		leftSwipeGesture.direction = .left
+		let doubleTapGesture = UITapGestureRecognizer(target: self, action: #selector(didDoubleTap))
+		doubleTapGesture.numberOfTapsRequired = 2
 
 		let pinchGesture = UIPinchGestureRecognizer(target: self, action: #selector(didPinch(_:)))
 
 		let panGesture = UIPanGestureRecognizer(target: self, action: #selector(didPan(_:)))
 
+		let longPeessGesture = UILongPressGestureRecognizer(target: self, action: #selector(didLongPress))
+
 		view.addGestureRecognizer(singleTapGesture)
 		view.addGestureRecognizer(pinchGesture)
+		view.addGestureRecognizer(longPeessGesture)
 
 		if viewModel.jumpGesturesEnabled {
-			view.addGestureRecognizer(rightSwipeGesture)
-			view.addGestureRecognizer(leftSwipeGesture)
+			view.addGestureRecognizer(doubleTapGesture)
+			singleTapGesture.require(toFail: doubleTapGesture)
```

Comment:
The "usability issue" that I mention is a long delay when tapping the _show_ the overlay when it isn't showing. I find it rather very annoying. My previous work had this exact logic of falling back gestures. I'm not against double tap, however I think that overall it's better to provide the option of double tap or swipe (as I actually prefer swipe). As long as there's the option I personally don't care about this delay, however it would be best to find the right logic to prevent the delay and mirror something like Youtube's double tap.

I think the lock gesture is nice, however I think it should ultimately be toggle-able from settings.

I'm okay to merge with:
- option for swipe/double tap
- option in Settings for gesture lock
- `longPeessGeature` -> `longPressGesture` spelling error

I know that I may be catering to my own preferences, however I just thought about how we would possibly want to implement 2x/3x/4x speed scrubbing which would be:
- tapping when you have swipe to skip
- swiping when you have tap to skip

Just a thought for work down the road.

Reply:
Oh, I got you.
I didn't check the UX properly because I only works on the simulator.
First of all, I will try to improve UX as much as possible in the current situation and add only the gesture lock option.
If UX cannot be improved, I will consider the swipe option.

Reply:
I removed the swipe because I wanted to control the slider with a pan gesture later. (like Infuse or etc..)

Reply:
That sounds great! I also think that in the future that we can create "gesture profiles" that are a combination of gestures from different products to appease the most people.